pkg/minikube/config/types.go

```diff
@@ -45,6 +45,7 @@ type MachineConfig struct {
 	HostOnlyCIDR        string // Only used by the virtualbox driver
 	HypervVirtualSwitch string
 	KvmNetwork          string             // Only used by the KVM driver
```

Comment:
Do you mind naming these members `KVM` instead of `Kvm`? 

Initialisms should be consistently cased: https://github.com/golang/go/wiki/CodeReviewComments#initialisms